WooCommerce/Classes/ViewModels/Order Details/OrderDetailsViewModel.swift

```diff
@@ -236,7 +236,12 @@ extension OrderDetailsViewModel {
             viewController.navigationController?.pushViewController(billingInformationViewController, animated: true)
         case .details:
             ServiceLocator.analytics.track(.orderDetailProductDetailTapped)
-            viewController.performSegue(withIdentifier: Constants.productDetailsSegue, sender: nil)
+            let identifier = ProductListViewController.classNameWithoutNamespaces
+            guard let productListVC = UIStoryboard.orders.instantiateViewController(identifier: identifier) as? ProductListViewController else {
+                return
```

Comment:
To aid debugging, would it be good to DDLogError or DDLogDebug in this unlikely event?

Reply:
Same comment for the other calls to `instantiateViewController`

Reply:
Thank you for the suggestion @allendav  👍  e55c39e3d1caccf371ca506f0c2b33932f4da4a0